utility/src/main/java/org/oppia/android/util/networking/NetworkConnectionUtil.kt

```diff
@@ -1,18 +1,21 @@
 package org.oppia.android.util.networking
 
+import androidx.annotation.StringRes
+import org.oppia.android.util.R
+
 /** Utility to get the current connection status of the device. */
 interface NetworkConnectionUtil {
 
   /** Enum to distinguish different connection statuses for the device. */
-  enum class ProdConnectionStatus : ConnectionStatus {
+  enum class ProdConnectionStatus(@StringRes var connectionName: Int) : ConnectionStatus {
```

Comment:
This should be a val.

Also suggest: "logName" to clarify where this name is used.

Reply:
Fixed